04/app.js

```diff
@@ -4,31 +4,42 @@ function init() {
     const boxElement = document.querySelector('.box');
     setBoxShadow(boxElement, '#000000');
 
-    const inputColor = document.querySelector('[type=color]')
-    inputColor.addEventListener('change', function(){
-       setBoxShadow(boxElement, this.value, inputRange.value*0.01); 
-       //console.log(this)
+    const inputColor = document.querySelector('[type=color]');
+    const inputRange = document.querySelector('[type=range]');
+    inputColor.addEventListener('change', handleChangeColor);
+    inputRange.addEventListener('change', handleChangeColor);
+    inputRange.addEventListener('mousemove', handleChangeColor);
+
+    function handleChangeColor(e){
+        //console.log(e, inputColor.value)
+        setBoxShadow(boxElement, inputColor.value, inputRange.value*0.01) // a co z parametrem w tym przypadku?
```

Comment:
nie do końca rozumiem pytanie, za każdym razem pobierasz aktualną wartość z obu elementów więc to jest to co potrzebujesz - coś nie działa?!